### PR TITLE
Update your first app on mac article for beta7

### PR DESCRIPTION
**Pull request by danroth27**
@ardalis @BrennanConroy @Rick-Anderson  Please review

### CODE REVIEW

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 1]
```diff
@@ -1,40 +1,62 @@
...
+  - `Publishing to Azure`_
 
 Setting Up Your Development Environment
 ---------------------------------------
 
-First, make sure you have :doc:`installed ASP.NET on your Mac OS X machine </getting-started/installing-on-mac>`. This step will include installing `Homebrew <http://brew.sh/>`_ and configuring DNVM. This article has been tested with DNX beta4. You can confirm that you are running the correct version of ``dnx`` by running the command ``dnvm list``. You should see ``default`` listed under the ``Alias`` column for ``1.0.0-beta4`` as shown:
-
-.. image:: your-first-mac-aspnet/_static/dnvm-list-beta4.png
-
+First, make sure you have :doc:`installed ASP.NET on your Mac OS X machine </getting-started/installing-on-mac>`. This step will include installing the .NET Execution Environment (DNX) and the .NET Version Manager (DNVM). You can check which DNX version you have active by running ``dnvm list``.
```
Comment (Rick-Anderson) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: You don't need </getting-started/installing-on-mac> on internal links - For example:
 :doc:`intro` describes the benefits that Tag Helpers provide.

Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: In this case I wanted to control the friendly text that gets displayed for the link

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 2]
```diff
@@ -1,40 +1,62 @@
...
+Create a folder for your solution and then add an ``src`` subfolder for your project source code.
+
+In your solution folder add a global.json file to indicate the DNX version:
+
+.. code-block:: json
+  :linenos:
+
+  {
+    "sdk": {
+      "version": "1.0.0-beta7"
```
Comment (BrennanConroy) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: Do we need to specify runtime here?

Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: It works fine without it.

**Comment by BrennanConroy** [comment]
Looks good

**Comment by Rick-Anderson** [comment]
:shipit:

**Pull request merged**